api/src/models/website.model.js

```diff
@@ -28,6 +28,14 @@ const websiteSchema = new mongoose.Schema(
       type: String,
       required: false,
     },
+    layout: {
```

Comment:
These should be enums
https://stackoverflow.com/questions/29299477/how-to-create-and-use-enum-in-mongoose

Reply:
Could you also remove the models that were no longer using. It’s getting confusing what we’re using and what we aren’t. Specifically everything in the editor directory I believe